components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -841,130 +839,55 @@
-        str(CONTAINER_CLP_HOME / "bin" / "node-14"),
-        str(container_webui_dir / "launcher.js"),
-        str(container_webui_dir / "main.js"),
-    ]
-    cmd = container_cmd + node_cmd
-    subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
-
-    logger.info(f"Started {component_name}.")
-
-
-def start_log_viewer_webui(
+def start_webui(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)